### PR TITLE
Tech stack documentation & onboarding links

### PR DESCRIPTION
**Pull request by avdonovan**
**1. Brief Summary of what this PR accomplishes (140 characters or less. If you find trouble describing what you are doing in this length, consider breaking the PR into multiple ones.)**

Adds a table in the onboarding docs that lists everything in the current tech stack, and some "getting started" links to help new team members learn.

**2. Link to Trello Ticket**

https://trello.com/c/NCdtyq8S/54-add-documentation-on-tech-stack-intro-to-each-individual-tech-choice

**3. More detailed description and other questions to address in code review**

Please add any other tutorials or links you've found helpful!  I'm also not sure if there are other technologies on the data gathering/analysis side that should be listed here.

**4. Remember to tag reviewers!**

### CODE REVIEW

File: onboarding/02_development_environment.md [hunk 1]
```diff
@@ -64,6 +64,24 @@ To download Postgres: http://oscg-downloads.s3.amazonaws.com/packages/PostgreSQL
...
+## Tech Stack
+
+Here are the technologies used in the project, along with some tutorials if you'd like to learn.
+
+| Tech | Version | Purpose | Getting Started |
+|------|---------|---------|-----------------|
+| Git | 2.4+ | Version control | [Udacity course](https://classroom.udacity.com/courses/ud775), [good comprehensive online book](https://git-scm.com/book/en/v2) |
+| Postgres | 9.6 | Database | [Tutorial](https://www.postgresql.org/docs/8.0/static/tutorial.html) |
+| SQL | |  Language used for database queries | [Tutorial from Postgres site](https://www.postgresql.org/docs/8.0/static/tutorial-sql.html) |
+| Python | v3 | Data Analysis & Webserver | [Tutorial](https://docs.python.org/3/tutorial/) |
```
Comment (VincentLa) [inline onboarding/02_development_environment.md]: Do you mind linking to Anaconda's distribution of Python? That's what we've been using so far and I still feel like that's the easiest way to get everyone on the same dev environment.

File: onboarding/02_development_environment.md [hunk 2]
```diff
@@ -64,6 +64,24 @@ To download Postgres: http://oscg-downloads.s3.amazonaws.com/packages/PostgreSQL
...
+| SciPy | | Python packages for data analysis | [Intro to Pandas](http://pandas.pydata.org/pandas-docs/stable/10min.html), [NumPy Tutorial](https://docs.scipy.org/doc/numpy-dev/user/quickstart.html) |
+| Jupyter | ? | Easily share Python analysis with code and results | [Quickstart guide](https://jupyter-notebook-beginner-guide.readthedocs.io/en/latest/) |
```
Comment (VincentLa) [inline onboarding/02_development_environment.md]: Yeah not sure if we have to worry about putting the version of jupyter

File: onboarding/02_development_environment.md [hunk 3]
```diff
@@ -64,6 +64,24 @@ To download Postgres: http://oscg-downloads.s3.amazonaws.com/packages/PostgreSQL
...
+| Django | 1.11.4 | Python webserver | [Getting started](https://docs.djangoproject.com/en/1.11/intro/) |
```
Comment (VincentLa) [inline onboarding/02_development_environment.md]: Not sure where to put the following, but this is another great feature of using Anaconda. With Anaconda, I can do `conda install django` and then put that in the `environment.yml` file. Then everyone can just import that file that everyone will have the same environment. We can even do that with R if we eventually do it. So it's not just python dependencies.

**Review by VincentLa — CHANGES_REQUESTED** [review]
This is great so far. Added minor comments. The other thing to keep in mind is if @cnwalker ends up splitting out our django and front end stuff in different repo than maybe the front end technologies go there

**Comment by avdonovan** [comment]
@VincentLa I made changes per your comments, please take a look.

File: onboarding/02_development_environment.md (hunk 1 above)
Comment (gregboyer) [reply onboarding/02_development_environment.md]: I've found this git reference to be really helpful as I got started: http://rogerdudler.github.io/git-guide/

**Review by VincentLa — APPROVED** [review]
lgtm!!

**Pull request merged**